### ISSUE TITLE
iframe visualization does not work

### ISSUE THREAD

**Issue by kkalyan** [airbnb/superset#2514]
- [x] I have checked the superset logs for python stacktraces and included it here as text if any
- [x] I have reproduced the issue with at least the latest released version of superset
- [x] I have checked the issue tracker for the same issue and I haven't found one similar

![image](https://cloud.githubusercontent.com/assets/391567/24453558/3cc120a2-143d-11e7-8437-342f76f541b0.png)

### Superset version
0.17.1

### Expected results
Iframe getting rendered. 

### Actual results
An error occurred while rendering the visualization: TypeError: e.render_template is not a function

### Steps to reproduce
Go to any data source
select iframe visualization 
tried with default youtube url and other website like google.com 

@mistercrunch @ascott

**Comment by yumminhuang** [comment]
I found the same problem on superset 0.17.3
Here is my solution. I use Markup visualization rather than iFrame, and set `Markup Type` as `html`. Put `<iframe>` tag in `code`.
```html
<iframe src="http://example.com" height="600" width="800" style="width:100%;"></iframe>
```

**Comment by kkalyan** [comment]
thanks @yumminhuang  for the workaround

**Comment by kkalyan** [comment]
Verified Fixed on 0.17.6
